scripts/post_install_script.bat

```diff
@@ -0,0 +1,22 @@
+@echo off
```

Comment:
I think we can merge this script with `install_script.bat`.

It can run right after pixi-pack, and before the deletion of the tar file.

Reply:
Done!